ldk/node/src/hostClients/browserClient.test.ts

```diff
@@ -0,0 +1,156 @@
+import { mocked } from 'ts-jest/utils';
+import createMockInstance from 'jest-create-mock-instance';
+import * as Services from '../grpc/browser_grpc_pb';
+import * as Messages from '../grpc/browser_pb';
+import { ConnInfo } from '../grpc/broker_pb';
+import { BrowserClient } from './browserClient';
+import { Session } from '../grpc/session_pb';
+import { Logger } from '../logging';
+import {
+  captureMockArgument,
+  createCallbackHandler,
+  createStreamingHandler,
+  createWaitHandler,
+  defaultConnInfo,
+  defaultSession,
+  identityCallback,
+} from '../jest.helpers';
+import { BrowserSelectedTextResponse } from './browserService';
+
+jest.mock('../grpc/browser_grpc_pb');
+
+const MockClientClass = mocked(Services.BrowserClient);
+
+const logger = new Logger('test-logger');
+
+describe('BrowserClient', () => {
+  let subject: BrowserClient;
+  let mockGRPCClient: jest.Mocked<Services.BrowserClient>;
+  let connInfo: ConnInfo.AsObject;
+  let session: Session.AsObject;
+
+  beforeEach(async () => {
```

Comment:
These could be pared down further into something that just wires up a useable `subject`. I've spent too much time on this though, so I'll just stop here, which is a decent improvement IMO.